provider/blob/tests/test_versions.rs

```diff
@@ -16,7 +16,7 @@ fn run_driver(exporter: BlobExporter) -> Result<(), DataError> {
     DatagenDriver::new()
         .with_keys([icu_provider::hello_world::HelloWorldV1Marker::KEY])
         .with_locales_and_fallback([LocaleFamily::full()], Default::default())
-        .export(&DatagenProvider::new_custom(), exporter)
+        .export(&icu_provider::hello_world::HelloWorldProvider, exporter)
```

Comment:
This change doesn't impact behavior; it just makes it so that this crate doesn't need to depend on the `provider` feature of icu_datagen

Reply:
Doesn't it need fallback data?

Reply:
It didn't have fallback data before and it doesn't have it now. This is the test that required me last time and again this time when refactoring the datagen driver to not run fallback when `LocaleFamily::full()` is used.